packages/core/src/stories/foundations/icons-gallery.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import formatHtmlPreview from '../formatHtmlPreview';
+import { iconsNames } from '../../components/icon/iconsArray';
+
+export default {
+  title: 'Foundations/Icons',
+  parameters: {
+    layout: 'fullscreen',
+    options: {
+      showPanel: false,
+      showToolbar: true,
+    },
+  },
+};
+
+const icons = iconsNames.map(
+  (icon) => `
+        <tds-block mode-variant="primary">  
+            <tds-icon name="${icon}" size="48"></tds-icon>  
+            <p class="tds-detail-05">${icon}</p>  
+        </tds-block>
+ `,
+);
+
+const Template = () =>
+  formatHtmlPreview(
+    `
+    <div style="display: grid; grid-template-columns: repeat(auto-fill, minmax(144px, 1fr)); gap: 16px; padding: 16px;">
```

Comment:
Could be nice to format the HTML string

Reply:
I tried some VS code plugins, but it get the same or worse, do you have some tip ?